app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -103,6 +114,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         binding = DataBindingUtil.inflate(inflater, R.layout.fragment_attendees, container, false);
 
         binding.setAttendees(attendeesPresenter.getAttendees());
+        setRetainInstance(true);
```

Comment:
Not working if I remove this too. I checked. This way at least it is not showing 404 not found. And we need to retain fragments. Please open an issue for this. This bug is even in master right now